src/Routes/Routes.tsx

```diff
@@ -18,12 +18,29 @@ function useLocationBackground() {
 
 export const Routes = () => {
   const { background, location } = useLocationBackground()
-  const { state } = useWallet()
+  const { connectDemo, state } = useWallet()
   const { appRoutes } = useBrowserRouter()
   const hasWallet = Boolean(state.walletInfo?.deviceId) || state.isLoadingLocalWallet
 
   return (
     <Switch location={background || location}>
+      <Route path='/demo'>
+        {() => {
+          const matchDemoPath = matchPath<{ rest: string }>(location.pathname, {
+            path: '/demo/:rest(.*)',
```

Comment:
This was an implementation example, I would name the rest of the path something meaningful, maybe `appRoute` (which would match the `appRoutes` naming on the next branch, happy to hear if you have a better terminology in mind!

Reply:
Also would go with `.+` since 0 or more chars will produce a 0-length `''` which technically evaluates as falsy, but might as well be safer

Reply:
`appRoute` seems fair enough to me.
https://github.com/shapeshift/web/pull/1709/commits/4a61d95fc7f89d0a94d326d26221035279c50d17 feels like a much nicer solution now.